docs/translations/api-docs-joy/menu/menu.json

```diff
@@ -11,6 +11,7 @@
     "onClose": "Triggered when focus leaves the menu and the menu should close.",
     "open": "Controls whether the menu is displayed.",
     "size": "The size of the component (affect other nested list* components because the <code>Menu</code> inherits <code>List</code>). To learn how to add custom sizes to the component, check out <a href=\"/joy-ui/customization/themed-components/#extend-sizes\">Themed components—Extend sizes</a>.",
+    "slots": "The components used for each slot inside the Popper. Either a string to use a HTML element or a component. See <a href=\"#slots\">Slots API</a> below for more details.",
```

Comment:
@hbjORbj Would this be fixed in this PR or the next one?

https://user-images.githubusercontent.com/18292247/230815711-4a327c57-c443-4440-b18a-e6f1142410fa.mov